### PR TITLE
Add tests to pkg/commands/git - Part 5

### PR DESCRIPTION
**Pull request by antham**

### CODE REVIEW

File: pkg/commands/git.go [hunk 1]
```diff
@@ -385,13 +385,11 @@ func (c *GitCommand) UnstageAll() error {
 
 // UnStageFile unstages a file
 func (c *GitCommand) UnStageFile(fileName string, tracked bool) error {
-	var command string
+	command := "git rm --cached %s"
 	if tracked {
-		command = "git reset HEAD "
-	} else {
-		command = "git rm --cached "
+		command = "git reset HEAD %s"
```
Comment (antham) [inline pkg/commands/git.go]: I was wondering here in which case a ```git reset HEAD``` doesn't work and you need to use ```rm --cached``` instead ?

**Comment by antham** [comment]
@jesseduffield could you review pls ?

File: pkg/commands/git.go (hunk 1 above)
Comment (jesseduffield) [reply pkg/commands/git.go]: In this case UnStageFile is probably not actually a good name for the function. This function will delete untracked files and unstage tracked files. That way the 'd' keybinding is a way to say 'remove this file from git status's output'

Comment (jesseduffield) [reply pkg/commands/git.go]: yes like `rm`

Comment (antham) [reply pkg/commands/git.go]: ok but to delete untracked files you have to do ```git clean -f``` so how it could work I don't understand ?

**Pull request merged**